ncm-nfs/src/main/perl/nfs.pod

```diff
@@ -44,17 +44,23 @@ change, then the volume will be remounted.
 
 =head1 NFS
 
-"/software/components/nfs/exports" = list(
-  nlist("path","/shared/path/",
-        "hosts",list("server*.example.org(no_root_squash)"))
-);
-
-# If the SE is exporting its disk, mount it on the worker nodes.
-"/software/components/nfs/mounts" = list(
-  nlist("device","foreign.example.org:/shared/path/",
-        "mountpoint","/mnt/foreign",
-        "fstype","nfs",
-        "options","rw")
-);
+    "/software/components/nfs/exports" = list(
```

Comment:
i consider assigning list like pure expert mode (as it resets any previously defined exports elsewhere). can we use `append(` here instead? it's an example afterall...